### ISSUE TITLE
[iOS] Wrong behaviour on biometric activation with wrong data after T&P acception

### ISSUE THREAD

**Issue by liliya-soroka** [safe-global/safe-ios#1090]
**Bug description**
1. After the T&P confirmation if the user can not activate biometric with valid FaceID or Touch ID he is offered  Enter passcode ,Cancel biometric activation or the suggestion to activate biometric is disappears in a few tries without any message
Also "Enter passcode" option doesn't lead to enter passcode during the biometric activation with wrong data. It's skipes the whole process

2. On passcode screen if the biometric is activated with the wrong data "Failed to activate biometrics" message appears
**Steps To Reproduce**
1. set up the app
2. accept T&P
3. set up a passcode
4. Use wrong data for Face ID or Touch ID a few times
**Current result** Message about wrong Face ID or Touch ID appears with two option: Enter the passcode and Cancel

5a. Click"Enter passcode"
**Current result** The message is closed, the user appears on the Assets page, the biometric option is disabled in the passcode settings
Note: If the biometric is activated on the passcode page and the biometric data can not be confirmed "Failed to activate biometric" message with an explanation is displayed for the user

5b: Try many times
**Current result** In some time the process of biometric activation is skipped without any information for the user. Assets page is displayed, biometric option is disabled

**Expected Result**
1. Use the message with try again instead of Enter passcode option for biometric activation 
2. Use "Failed to activate biometric" message if the biometric can not be activated because of wrong data - the same as on passcode settings page when the biometric is activated with wrong data  

**Device & App version**
 - Device/OS: Huawei P30
 - App version: 2.14.0 (7)
 
**Environment**
prod , mainnet
**Additional info**
(Optional)

**Comment by liliya-soroka** [comment]
My opinion: it's not critical for the release, as It's an additional case and the biometric option is not active if the wrong data is used for faceID and Touch ID

**Comment by DmitryBespalov** [comment]
- Another case is with the blocking of the touch/Face ID and need to enter device passcode
- After update from previous version, device passcode is disabled, then app will have missing "Touch ID" setting even after enabling passcode.

**Comment by liliya-soroka** [comment]
The clean app install is required if the FaceID or TouchID was blocked by the system because of biometrics activation with wrong data a few times (3-5) or wait till it will be unlocked by the system ( not sure how long).

**Comment by sche** [comment]
@liliya-soroka did you check it on the latest RC?

**Comment by liliya-soroka** [comment]
Verified:
1. the user is offered to enter the device passcode if the wrong biometric is used many times and the system blockes biometric usage
2. The user is informed that biometric is blocked and suggested to try late during biometric activation with wrong biometric data ( different from the already set up on device)

The difference with the previous version:
there is no link on the app settings on the device to enroll FaceID/Touch ID in device settings before it will be activated in app - the separate issue will be created and hendalled late

app version 2.14.0 (10)
